ASAudioKit/ASAudioRecorder/ASAudioRecorder.swift

```diff
@@ -0,0 +1,30 @@
+import AVFoundation
+
+class ASAudioRecorder {
+    private var audioRecorder: AVAudioRecorder? = nil
+    
+    func startRecording(url: URL) {
+        let settings = [
+            AVFormatIDKey : Int(kAudioFormatMPEG4AAC),
+            AVSampleRateKey : 12000,
+            AVNumberOfChannelsKey : 1,
+            AVEncoderAudioQualityKey : AVAudioQuality.high.rawValue
```

Comment:
```suggestion
            AVFormatIDKey: Int(kAudioFormatMPEG4AAC),
            AVSampleRateKey: 12000,
            AVNumberOfChannelsKey: 1,
            AVEncoderAudioQualityKey: AVAudioQuality.high.rawValue
```
Lint 에러입니다. 띄어쓰기를 없애는 것이 좋아 보여요.
딕셔너리는 [Key: Value] 형식으로 써주세용